README.md

```diff
@@ -27,17 +25,35 @@ To run the site locally:
    ```
 4. Open http://localhost:8080
 
-### Accessiblity Scanning
+### Codespaces
+[Codespaces](https://github.com/features/codespaces) allows you to set up a development environment easily and without dependencies on a local machine.
+
+#### Start a GitHub Codespace
+1. From this repo, Click "Code" and then the "Codespaces" tab.
+2. Click the "+" icon, agree to the terms.
+3. You should then see the repo interface. At the bottom, check the branch you are currently viewing.
+
+#### Start the application and preview the site
+1. Give this a few minutes to warm up. At first, it will have a message "Welcome to Codespaces."  If you wait a bit, you'll see it recognize the environment and start to install libraries for you.
+2. Once it finishes installing the libraries, click on the "terminal" and type ```npm start```.
```

Comment:
`npm start` works well for content changes, but style changes won't show up until we restart the server. For those changes it would probably be better to use `npm run dev`